### PR TITLE
FailoverClustering Bulk Fixup (3/4) - Line formatting updates

### PR DESCRIPTION
**Pull request by robinharwood**
Bulk reflowed text and header formatting update before applying further quality updates.

This will be followed by separate PRs for 
	1. Text formatting updates
		a. Bulk formatting checklist update before applying further quality updates:
			- cmdlet name - Change bolds to code spans
			- cmdlet/vars - Change bar ecmdlet/var to code spans
			- parameters - change italic formatting to bold
			- parameters - change code span formatting to bold
			- change italics to use underscores
			- change bold-underscores to asterisk
			- change bold keystrokes to kbd
			- Use contractions
			- Fix indented code blocks
			- Find bad syntax reflows
	2. Update example formatting
		a. Bulk updating example formatting before content updates

### CODE REVIEW

**Comment by robinharwood** [comment]
Hi @michaeltlombardi / @sdwheeler,

I believe this PR is release ready if you would be kind enough to review and if appropriate sign-off/merge 😊

The main update was to reflow and fix any line return issues.

Please let me know if you have any questions or if I need to apply any other updates.

Thanks Robin

File: docset/winserver2022-ps/failoverclusters/Set-ClusterParameter.md [hunk 1]
```diff
@@ -11,55 +11,69 @@ title: Set-ClusterParameter
...
 
 ## DESCRIPTION
-The **Set-ClusterParameter** cmdlet controls specific properties of an object in a failover cluster, such as a resource, a group, or a network. 
 
- -- For a disk resource, you can set the disk signature or GUID of a disk, and turn maintenance on or off for that disk. 
+The **Set-ClusterParameter** cmdlet controls specific properties of an object in a failover cluster,
+such as a resource, a group, or a network.
+
+ -- For a disk resource, you can set the disk signature or GUID of a disk, and turn maintenance on
+ or off for that disk.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/failoverclusters/Set-ClusterParameter.md:46]: - For a disk resource, you can set the disk signature or GUID of a disk, and turn maintenance on
  or off for that disk.

File: docset/winserver2022-ps/failoverclusters/Set-ClusterParameter.md [hunk 2]
```diff
@@ -11,55 +11,69 @@ title: Set-ClusterParameter
...
 
- -- For a Network Name resource, you can set DNS-related information about the resource. 
+ -- For a Network Name resource, you can set DNS-related information about the resource.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/failoverclusters/Set-ClusterParameter.md:48]:
```suggestion
- For a Network Name resource, you can set DNS-related information about the resource.
```

File: docset/winserver2022-ps/failoverclusters/Set-ClusterParameter.md [hunk 3]
```diff
@@ -11,55 +11,69 @@ title: Set-ClusterParameter
...
 
- -- For an IP address resource, you can set DHCP-related information about the IP Address resource. 
+ -- For an IP address resource, you can set DHCP-related information about the IP Address resource.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/failoverclusters/Set-ClusterParameter.md:50]:
```suggestion
- For an IP address resource, you can set DHCP-related information about the IP Address resource.
```

File: docset/winserver2022-ps/failoverclusters/Set-ClusterParameter.md [hunk 4]
```diff
@@ -11,55 +11,69 @@ title: Set-ClusterParameter
...
 
- -- For resources used by virtual machines, you can set details about the settings for the virtual machines.
+ -- For resources used by virtual machines, you can set details about the settings for the virtual
+ machines.
```
Comment (michaeltlombardi) [inline docset/winserver2022-ps/failoverclusters/Set-ClusterParameter.md:53]:
```suggestion
- For resources used by virtual machines, you can set details about the settings for the virtual
  machines.
```

**Review by michaeltlombardi — CHANGES_REQUESTED** [review]
Minor notes for syntax that breaks in-console help.

**Review by michaeltlombardi — APPROVED** [review]

**Pull request merged**